### PR TITLE
Feature/import flow with session templates

### PR DESCRIPTION
**Pull request by AkhileshNegi**
- Added support to import/export flow with templates
- Added testcases

### CODE REVIEW

File: lib/glific/flows.ex [hunk 1]
```diff
@@ -742,7 +743,7 @@ defmodule Glific.Flows do
                }),
              {:ok, _flow_revision} <-
                FlowRevision.create_flow_revision(%{
-                 definition: flow_revision["definition"],
+                 definition: clean_flow_with_template(flow_revision["definition"]),
```
Comment (pankaj-ag) [inline lib/glific/flows.ex]:
```suggestion
                 definition: clean_flow_for_templates(flow_revision["definition"]),
```

Also, is it for HSM or interactive, please add that name as well.

File: test/glific_web/schema/flow_test.exs [hunk 2]
```diff
@@ -9,6 +9,7 @@ defmodule GlificWeb.Schema.FlowTest do
     Flows.Flow,
     Flows.FlowRevision,
     Groups,
+    # Messages,
```
Comment (pankaj-ag) [inline test/glific_web/schema/flow_test.exs]: Let's use or remove

**Review by pankaj-ag — CHANGES_REQUESTED** [review]
Also, do you think, in case of HSM we can create a dummy template object with EXPIRE_UUID while importing the flow?

**Review by pankaj-ag — APPROVED** [review]

**Comment by pankaj-ag** [comment]
We can merge when we pass all the tests

File: lib/glific/flows.ex (hunk 1 above)
Comment (AkhileshNegi) [reply lib/glific/flows.ex]: made requested changes

File: test/glific_web/schema/flow_test.exs (hunk 2 above)
Comment (AkhileshNegi) [reply test/glific_web/schema/flow_test.exs]: removed

**Pull request merged**